Okta.Xamarin/Okta.Xamarin/OktaXamarinUserAgent.cs

```diff
@@ -0,0 +1,15 @@
+﻿// <copyright file="OktaXamarinUserAgent.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Reflection;
+
+namespace Okta.Xamarin
+{
+	public class OktaXamarinUserAgent
+    {
+        public static string Value => new Lazy<string>(() => $"Okta-Xamarin-Sdk/{Assembly.GetExecutingAssembly().GetName().Version}").Value;
```

Comment:
Every time this Value is accessed a new `Lazy<string>` object is being created and initialized. I don't think this was the intention here.

Reply:
😆 Oops!

Reply:
Fixed: https://github.com/okta/okta-oidc-xamarin/pull/35/commits/23bcf53bed1bed7b748509d27efa60088276b98b